### DISCUSSION TITLE
Danish translation

### DISCUSSION THREAD

**Discussion by ERYpTION** [Helium314/SCEE#468] [General]
Will you add Danish to Weblate? I can help with the translation.

**Comment by Helium314** [comment]
I thought any user could add a language. Doesn't this work for you?

**Reply by ERYpTION** [reply]
yes, sorry it did.

**Reply by Helium314** [reply]
Ah, good. I was just trying to find the option to add a language on Weblate and actually couldn't...
Could you describe where it is, then I'll add it to the translation section of the readme.

**Comment by ERYpTION** [comment]
First I had to create a Codeberg-account. Then when logging in to Weblate, Danish was at the top of the language list as an option to add. Not sure how it knew to suggest that but I then simply had to click the + to add it.

**Reply by Helium314** [reply]
Ah, so you probably can only add languages that you have selected in the _Language_ menu on top.

**Reply by ERYpTION** [reply]
I didn't get around to it as quickly as I expected but Danish translation is now done!

**Reply by Helium314** [reply]
Great! It will be (semi)automatically added in the next release.

**Reply by ERYpTION** [reply]
Thanks. I will probably do a few adjustments once I see the translations in the app.

**Reply by ERYpTION** [reply]
By the way, I couldn't find the strings for the height limit quest when there is no sign, i.e. "Is there easily enough space for even the tallest trucks ..." etc.

**Reply by Helium314** [reply]
The translations contain only SCEE-exclusive strings. The one you mention is in StreetComplete.

**Reply by ERYpTION** [reply]
There are no untranslated strings in SC. Perhaps it's because SC no longer has this quest (doesn't ask for more information when no sign)?

**Reply by Helium314** [reply]
Ah, I had re-added this to the SC strings.xml.
The idea was that once I find time I could simply extract the translated SC strings from an old version and move them to SCEE. But too much to do all the time...

**Reply by ERYpTION** [reply]
In addition to the truck-text there is also still some untranslatable text in the Data management section : "Use temp logger. Log will not be persisted in database .... "

Thanks.
